### PR TITLE
Submission to list PoolTogether in Valora

### PR DESCRIPTION
**Pull request by gsalaiz-Valora**
> To make a submission to the Dapps Page, please complete the task lists
> below; otherwise remove them.

Your Dapp:

- [X] connects to Valora via WalletConnect V1 or DAppKit;
- [X] must be mobile-first or mobile responsive;
- [X] is in one of the existing categories; and
- [X] includes a link to user documentation.

### CODE REVIEW

File: locales/base.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "ethichub": "Lend affordable capital to unbanked farmers",
     "mezumo": "Buy tokens over time using a simple strategy called Dollar-Cost Averaging (DCA)",
     "uniswap": "Swap tokens or earn yield by providing liquidity",
-    "pixpress": "Swap any amount of any crypto assets with others peer-to-peer and earn rewards"
+    "pixpress": "Swap any amount of any crypto assets with others peer-to-peer and earn rewards",
+    "pooltogether": "PoolTogether is a prize savings protocol, enabling you to win by saving"
```
Comment (MuckT) [inline locales/base.json:46]: Can we change to something that doesn't included that name of the Dapp in the description and matches the tone of other Dapp descriptions? "Save while participating in prize raffles"

File: src/valora-dapp-list.json [hunk 2]
```diff
@@ -239,6 +239,13 @@
...
       "categoryId": "exchanges",
       "url": "https://pixpress.pixelava.space"
+    },
+    {
+      "name": "PoolTogether",
+      "id": "pooltogether",
+      "categoryId": "lend",
+      "url": "https://v3.pooltogether.com/pools?filter=celo"
     }
+
```
Comment (MuckT) [inline src/valora-dapp-list.json:249]: nit: extra space.

**Review by MuckT — DISMISSED** [review]
LGTM after the description change and extra space is removed.

File: locales/base.json (hunk 1 above)
Comment (gsalaiz-Valora) [reply locales/base.json:46]: agreed!

**Review by jeanregisser — APPROVED** [review]

**Pull request merged**